rasa/core/brokers/sql.py

```diff
@@ -46,7 +46,9 @@ def __init__(
             dialect, host, port, db, username, password
         )
 
-        logger.debug(f"SQLEventBroker: Connecting to database: '{engine_url}'.")
+        logger.debug(
+            f"SQLEventBroker: Connecting to database: '{engine_url.__to_string__()}'."
```

Comment:
this will not work if a URL has been supplied explicitly as an environment variable - `get_db_url()` might return a string